tyk-docs/assets/scss/_side-menu.scss

```
@@ -187,12 +188,13 @@ Tree View
       background-size: 12px;
       inline-size: 100%;
       block-size: var(--caret-size);
+      block-size:35px;
```

```suggestion
      block-size: 30px;
```
Can we make the caret size smaller as we reduced the text size of side nav options during the restructuring?
Currently the caret looks the same size of text.